text_formatter.go

```diff
@@ -160,7 +163,7 @@ func (f *TextFormatter) Format(entry *Entry) ([]byte, error) {
 		timestampFormat = defaultTimestampFormat
 	}
 	if f.isColored() {
-		f.printColored(b, entry, keys, timestampFormat)
+		f.printColored(b, entry, keys, nil, timestampFormat)
```

Comment:
Should we give the new data array as parameter here ?

Reply:
Fixed.